src/main/resources/view/HelpWindow.fxml

```diff
@@ -19,6 +20,7 @@
       <stylesheets>
         <URL value="@HelpWindow.css" />
       </stylesheets>
+      <ScrollPane fx:id="helpMessageScrollPane">
```

Comment:
Try this <ScrollPane fx:id="helpMessageScrollPane" hbarPolicy="NEVER" fitToWidth="true">